packages/tupaia-web/src/pages/Project.tsx

```diff
@@ -21,10 +13,10 @@ export const Project = () => {
   // const { projectCode, entityCode, '*': dashboardCode } = useParams();
 
   return (
-    <Container>
-      <MapLayout />
-      <Sidebar />
+    <>
+      <TabLayout />
```

Comment:
Same here, maybe this could be MobileTabLayout to make it really obvious to new devs to the project